src/store/CategoricalDisplayStore.js

```diff
@@ -214,9 +214,12 @@ export default class CategoricalDisplayStore {
 
     @computed get filteredItems() {
         let filteredItems = cloneCategories(this.getCategoricalItems());
+        const activeItmes = this.activeCategories.length
+            ? this.activeCategories
+            : [(this.getCurrentActiveCategory ? this.getCurrentActiveCategory() : 'favorite')];
```

Comment:
Because we set the render of component for when a user opens it, so `this.activeCategories` not initial unless the component gets opens. and `filteredItems` render after component load.